sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -331,22 +322,20 @@ export class EventHubClient {
    * @returns {EventHubClient} - An instance of the eventhub client.
    */
   static createFromConnectionString(connectionString: string, path?: string, options?: ClientOptions): EventHubClient {
-    if (!connectionString || (connectionString && typeof connectionString !== "string")) {
-      throw new Error("'connectionString' is a required parameter and must be of type: 'string'.");
-    }
```

Comment:
Lets not change any of the parameter checks being made in this PR.
We will tackle this as part of https://github.com/Azure/azure-sdk-for-js/issues/2660

If we make changes to the errors being returned, that can be considered as breaking change.
We want to make all non breaking changes first, may be release an update and then do minimal breaking changes